services/app-api/src/resolvers/rate/undoWithdrawRate.test.ts

```diff
@@ -379,7 +379,7 @@ describe('undoWithdrawRate', () => {
                 bodyHTML: expect.stringContaining(contractBName),
             })
         )
-    }, 10000)
```

Comment:
observation: we need to bump the overall timeout on backend tests I think. I see we randomly extend to 15000, 20000, 30000 something places its all scattered/random. I would assign a variable to this `EXTENDED_TIMEOUT` thats shared or just bump the overall settings on the test config